docs/kqueen.cli_gcloud_examples.rst

```diff
@@ -0,0 +1,137 @@
+Kqueen and Google cloud from scratch
+------------------------------------
+
+Create the Google Cloud account and start with `gcloud`:
+
+https://cloud.google.com/pubsub/docs/quickstart-cli
+
+Login to Google Cloud
+
+.. code-block:: bash
+
+   $ cd /tmp/
```

Comment:
remove $

Reply:
I would like to keep the "$" in front of every command. This is a good practice from other projects (http://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html) and it helps to automate running of the commands (byt CI if necessary) and differentiate commands from it's output:

```
$ ls
api.yml  index.rst  kqueen.blueprints.api.rst  kqueen.cli_api_examples.rst     kqueen.engines.rst  kqueen.storages.rst
conf.py  key.json   kqueen.blueprints.rst      kqueen.cli_gcloud_examples.rst  kqueen.rst          Makefile
```

Let me know what you think about it.